versions.json

```diff
@@ -4,23 +4,22 @@
       "path": "4.0.0-preview1/scala2.13-java17-python3-ubuntu",
       "tags": [
         "4.0.0-preview1-scala2.13-java17-python3-ubuntu",
-        "4.0.0-preview1-java17-python3",
-        "4.0.0-preview1-java17",
-        "python3-java17"
+        "4.0.0-preview1-python3",
+        "4.0.0-preview1"
```

Comment:
shall we tag the "all-in-one" image with this spark version only name?

Reply:
Or we think it's better to exclude R support by default?

Reply:
I'm neutral on this. As I mentioned only had some [short discussion](https://docs.google.com/document/d/1nN-pKuvt-amUcrkTvYAQ-bJBgtsWb9nAkNoVNRM2S2o/edit?disco=AAAAf2TyFr0) on it.

cc @zhengruifeng @HyukjinKwon @gengliangwang any idea?

[1] https://docs.google.com/document/d/1nN-pKuvt-amUcrkTvYAQ-bJBgtsWb9nAkNoVNRM2S2o/edit?disco=AAAAf2TyFr0

Reply:
if this was the first version, I might have preferred to have all-in-one to have `4.0.0-preview1`
but it's been like that already so I think it's fine

Reply:
+1

Reply:
Actually, `4.0.0-preview1` haven't been tagged in the published image yet.

So, we can move `4.0.0-preview1` to `4.0.0-preview1/scala2.13-java17-python3-r-ubuntu`. that means the `apache/spark:4.0.0-preview1/scala2.13-java17-python3-r-ubuntu` will have a new alias tag `4.0.0-preview1`.

I can add the `4.0.0-preview1`.